pam_authz/docker/docker-compose.yaml

```diff
@@ -12,18 +12,31 @@ services:
       - "sudo.rego"
       - "io.directory.users:/users.json"
       - "io.vcs.contributors:/contributors.json"
+      # Prompt policies.
+      - "display.rego"
+      - "authz.rego"
+      - "pull.rego"
     volumes:
       - ./policy/ssh.rego:/ssh.rego
       - ./policy/sudo.rego:/sudo.rego
       - ./policy/users.json:/users.json
       - ./policy/contributors.json:/contributors.json
+      # Prompt policies.
+      - ./policy/display.rego:/display.rego
+      - ./policy/authz.rego:/authz.rego
+      - ./policy/pull.rego:/pull.rego
 
   frontend:
     image: openpolicyagent/demo-pam
     ports:
       - "2222:22"
     volumes:
       - ./etc/frontend_host_id:/etc/host_identity.json
+    environment:
+      - TEST=TODO_REMOVE
```

Comment:
Is this needed? If so, should rename to make clear that it's useful and not a debugging leftover

Reply:
Removed, thanks!